scripts/build-install.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+make build
+mv $GOPATH/bin/terraform-provider-namecheap ~/.terraform.d/plugins/terraform-provider-namecheap
+chmod +x ~/.terraform.d/plugins/terraform-provider-namecheap
```

Comment:
I'll remove this if you want. I run this from root.

Reply:
I've got a symlink for the same reason.

```
$ ls -l ~/.terraform.d/plugins/
total 0
lrwxr-xr-x  1 adam  staff  49 Dec  4 11:58 terraform-provider-namecheap -> /Users/adam/code/bin/terraform-provider-namecheap
```

Reply:
Should I go with the symlink or is this script ok? This really should be in the makefile..

Reply:
I'd say symlink unless a file already exists at `~/.terraform.d/plugins/terraform-provider-namecheap`